package.json

```diff
@@ -11,7 +11,7 @@
     "concourse:deploy-pipeline": "fly -t cr-cicd-engineering set-pipeline -p serverless-starter-app -c concourse/pipeline.yml -l concourse/private.yml",
     "concourse:login": "fly --target cr-cicd-engineering login --concourse-url https://ci.services.comicrelief.com --team-name engineering",
     "concourse:sync": "sudo fly -t ci sync",
-    "offline": "sls offline --apiKey api-key",
+    "offline": "sls offline --param=\"apiKey=api-key\"",
```

Comment:
We can remove the `apiKey` parameter with the new version of `serverless-offline`. It was required previously because `serverless-offline` didn't support configuring API keys and generated a random one on each run (so, in order to test, we have to override that with something known). The new version correctly loads API keys from `serverless.yml:provider.apiGateway.apiKeys`.

Reply:
Great spot - thanks.

Reply:
Actually, I don't fully understand this. There is no `provider.apiGateway` entry in `serverless.yml`. Do I add this through `.env` or something?

Reply:
Oh, actually in the `serverless.yml` file - I see

Reply:
Also, the starter app doesn't have any private endpoints or API keys configured! It's a case of add it if you need it when building from the starter app.

Reply:
But we want the starter-app to include examples of the kind of things that we typically need when building sls apps, and since we pretty much always use `apiKeys` for authorisation does it seem like a good idea to include a template for doing that here?

Reply:
I feel like I'm being a pain, but would prefer if you remove, since it's not directly related to the Serverless update and adds API Gateway infrastructure to the deployment.